static/js/global.js

```diff
@@ -0,0 +1,48 @@
+var browserLocale
+
+if (!localStorage.getItem('lang')) {
+  var nav = window.navigator
+  browserLocale =
+    nav.language.split('-')[0] || nav.language || nav.userLanguage || ''
+  localStorage.setItem('lang', browserLocale)
```

Comment:
Only store if the user has specifically selected a language.

Reply:
ok